kind.sh

```diff
@@ -20,6 +20,7 @@ set -o pipefail
 
 DEFAULT_KIND_VERSION=v0.6.1
 DEFAULT_CLUSTER_NAME=chart-testing
+DEFAULT_KUBECTL_VERSION=v1.17.0
```

Comment:
I wouldn't prefix it with `DEFAULT_` because that implies it can be overridden which is not the case. We could make it configureable but I'm not sure if this is necessary.